cfgov/unprocessed/js/modules/transition/BaseTransition.js

```diff
@@ -198,9 +203,9 @@ function BaseTransition( element, classes ) { // eslint-disable-line max-stateme
   this.setElement = setElement;
 
   // Public static constants.
-  BaseTransition.BEGIN_EVENT = 'transitionBegin';
-  BaseTransition.END_EVENT = 'transitionEnd';
-  BaseTransition.NO_ANIMATION_CLASS = 'u-no-animation';
+  this.BEGIN_EVENT = BEGIN_EVENT;
+  this.END_EVENT = END_EVENT;
+  this.NO_ANIMATION_CLASS = NO_ANIMATION_CLASS;
```

Comment:
Confirmed fixed by moving constants outside of the constructor and removing redundant firing of `transitionEnd` --> https://github.com/cfpb/cfgov-refresh/pull/1641